h2o-automl/src/main/java/ai/h2o/automl/modeling/StackedEnsembleStepsProvider.java

```diff
@@ -130,6 +138,71 @@ protected boolean isStackedEnsemble(Key<Model> key) {
                         return stack(_algo+"_AllModels", getBaseModels(), true);
                     }
                 },
+                new StackedEnsembleModelStep("monotone", DEFAULT_MODEL_TRAINING_WEIGHT, aml()) {
+                    { _description = _description+" (built using monotonically constrained AutoML models)"; }
+
+                    boolean hasMonotoneConstrains(Key<Model> modelKey) {
+                        Model model = DKV.getGet(modelKey);
+                        try {
+                            KeyValue[] mc = (KeyValue[]) model._parms.getClass()
+                                    .getField("_monotone_constraints")
+                                    .get(model._parms);
+                            return mc != null && mc.length > 0;
+                        } catch (NoSuchFieldException | IllegalAccessException e) {
+                            return false;
+                        }
```

Comment:
please try to use `PojoUtils` whenever you can for reflection.
`PojoUtils.getFieldValue()` should do the job here.

Reply:
or `ReflectionUtils`... whatever

Reply:
Thanks for the suggestion!